services/skus/service.go

```diff
@@ -1937,3 +1940,7 @@ type tlv1CredPresentation struct {
 	IssuedAt  string `json:"issuedAt"`
 	ExpiresAt string `json:"expiresAt"`
 }
+
+func ptrTo[T any](v T) *T {
+	return &v
+}
```

Comment:
@clD11 had removed it in the previous PRs, and I was planning on updating https://github.com/brave-intl/bat-go/pull/2183, but it was merged without asking, so it's making a come back.